scripts/testing/setup-mock-test-env.py

```diff
@@ -343,13 +383,15 @@ def setup_mock(mock_command):
 
     if ns.init:
         setup_mock(mock_cmd)
-        mock_init_run = True
-        if ns.install:
-            install_packages_to_mock(mock_cmd, ns.install)
+        if ns.install_pip is None:
+            install_required_pip_packages(mock_cmd)
```

Comment:
I am a little confused. Why don't we install the required pip packages in `setup_mock`?

Reply:
Because this way it is done on one place and you can easily ignore the default set.